templates/focus-ios.xliff

```diff
@@ -20,14 +17,44 @@
       <trans-unit id="NSMicrophoneUsageDescription">
         <source>This lets you take and upload videos.</source>
       </trans-unit>
+      <trans-unit id="NSPhotoLibraryAddUsageDescription">
+        <source>This lets you save and upload photos.</source>
+      </trans-unit>
       <trans-unit id="NSPhotoLibraryUsageDescription">
         <source>This lets you save and upload photos.</source>
       </trans-unit>
     </body>
   </file>
+  <file original="Blockzilla/Intro.strings" source-language="en" datatype="plaintext">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="9.0" build-num="9A235"/>
+    </header>
+    <body>
+      <trans-unit id="Intro.Slides.Search.Description">
```

Comment:
Are we sure all these are proper "Intro strings" ID's (so not only this one, but the rest for the Intro strings as well)? They look very different from the format used elsewhere throughout the file. May need @flodolo's input here just to confirm though, because I don't know how big a deal it is

Reply:
As long as they contain an indication of the role (description, title, button, etc.), they're OK